### PR TITLE
React routes into basic-feature-routes

### PR DESCRIPTION
**Pull request by jwu910**

### CODE REVIEW

File: src/components/Main.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        <Route path="/dashboard" component={Dashboard} />
+        <Route path="/wishlist" component={WishlistPublic} />
+        <Route path="/aboutus" component={AboutUs} />
+        <Route path="/secure/wishlist" component={WishlistPrivate} /> {/* Secure */}
+        <Route path="/secure/pairing" component={Pairing} /> {/* Secure */}
+        <Route path="/secure/messaging" component={Messaging} /> {/* Secure */}
+        <Route path="/secure/org" component={SecureOrg} /> {/* Secure */}
+      </Switch>
+      <footer>
+        <a href="/">
```
Comment (phuchle) [inline src/components/Main.js]: I noticed this should be a Link component.

**Review by phuchle — CHANGES_REQUESTED** [review]

File: src/components/Main.js (hunk 1 above)
Comment (jwu910) [reply src/components/Main.js]: the anchor?

Comment (phuchle) [reply src/components/Main.js]: Yup.  Should link to root.

**Review by phuchle — APPROVED** [review]

**Review by rainrivas — APPROVED** [review]
We will need to beefup the tests in development ( or after the prorotype milestone ) to test more meaningful things but we are good to go for now

**Comment by jwu910** [comment]
Commenting from my phone not sure where this is ending up. 

But yeah efrain need to beef up tests. I saw some of the Jest tests can search for elements after a component is registered. Lots to learn still. 

_Sent from my Samsung SAMSUNG-SM-G930A using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Review by AlChiu — APPROVED** [review]

**Review by psyduc — APPROVED** [review]

**Pull request merged**